### PR TITLE
Add: drivedlgo

### PR DESCRIPTION
**Pull request by semisoft0072**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by HUMORCE** [comment]
Does not meet the criteria of this bucket.

> Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 100 stars and/or 50 forks)
> 
> https://github.com/ScoopInstaller/Extras/blob/master/.github/ISSUE_TEMPLATE/package-request.yml

**Comment by semisoft0072** [comment]
@HUMORCE ooh sorry for that 
Then which bucket is used to add this app is there any other bucket for that?

**Comment by HUMORCE** [comment]
> Then which bucket is used to add this app is there any other bucket for that?

No, you can keep this PR open and ping when the criteria are met.

You have put it in your personal bucket and it is searchable by other users: https://scoop.sh/#/apps?q=%22https%3A%2F%2Fgithub.com%2Fsemisoft0072%2Fscoop-apps%22&s=0&d=1&o=false

**Comment by semisoft0072** [comment]
oh okay thanks.

File: bucket/drivedlgo.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+  "version": "1.6.7",
+  "description": "A Minimal Google Drive Downloader Written in Go.",
+  "homepage": "https://github.com/JaskaranSM/drivedlgo",
+  "license": "MIT License",
```
Comment (rasa) [inline bucket/drivedlgo.json:5]: This should be simply `MIT`

Comment (semisoft0072) [reply bucket/drivedlgo.json]: Done thanks for info.

**Pull request closed without merging**